autopts/ptsprojects/boards/nrf53.py

```diff
@@ -40,13 +40,22 @@ def build_and_flash(zephyr_wd, board, debugger_snr, conf_file=None, *args):
     if conf_file and conf_file != 'default' and conf_file != 'prj.conf':
         bttester_overlay += f';{conf_file}'
 
+    host_overlay = "btstester_overlay.conf"
+    check_call(f'echo CONFIG_BT_BUF_ACL_RX_SIZE=100 > {host_overlay}'.split(),
+               cwd=tester_dir)
+    bttester_overlay += f';{host_overlay}'
+
     cmd = ['west', 'build', '-b', board, '--', f'-DOVERLAY_CONFIG=\'{bttester_overlay}\'']
     check_call(cmd, cwd=tester_dir)
     check_call(['west', 'flash', '--skip-rebuild', '--recover', '-i', debugger_snr], cwd=tester_dir)
 
     controller_overlay = 'bttester_hci_rpmsg_overlay.conf'
     check_call(f'echo CONFIG_BT_CTLR_CONN_ISO_LOW_LATENCY_POLICY=y > {controller_overlay}'.split(),
                cwd=controller_dir)
+    check_call(f'echo CONFIG_BT_CTLR_DATA_LENGTH_MAX=100 >> {controller_overlay}'.split(),
+               cwd=controller_dir)
+    check_call(f'echo CONFIG_BT_BUF_ACL_RX_SIZE=100 >> {controller_overlay}'.split(),
+               cwd=controller_dir)
 
     cmd = ['west', 'build', '-b', 'nrf5340dk_nrf5340_cpunet', '--',
            f'-DOVERLAY_CONFIG=\'nrf5340_cpunet_iso-bt_ll_sw_split.conf;{controller_overlay}\'']
```

Comment:
and add the new net core overlay file:
```suggestion
    cmd = ['west', 'build', '-b', 'nrf5340dk_nrf5340_cpunet', '--',
           f'-DOVERLAY_CONFIG=\'nrf5340_cpunet_iso-bt_ll_sw_split.conf;'
           f'../../../tests/bluetooth/tester/nrf5340_hci_rpmsg_cpunet.conf\'']
```